cmd/earthly/main.go

```diff
@@ -295,7 +295,15 @@ func main() {
 		ctxTimeout, cancel := context.WithTimeout(ctx, time.Millisecond*500)
 		defer cancel()
 		displayErrors := app.verbose
-		analytics.CollectAnalytics(ctxTimeout, app.apiServer, displayErrors, Version, getPlatform(), GitSha, app.commandName, exitCode, time.Since(startTime))
+		cc, err := cloud.NewClient(app.apiServer, app.sshAuthSock, app.authToken, app.console.Warnf)
+		if err != nil && displayErrors {
```

Comment:
If there's an error but the user is NOT in verbose mode, this will fall through to the `else` below and will use `cc` which will be nil.

You'll need to break them up like:
```
if err != nil {
    if displayErrors {
        app.console.Warnf("unable to start cloud client: %s", err)
    }
} else { ...
```

Reply:
oh yeah definitely, thanks for catching my mistake :)